Product/Production/Common/CONNECTCoreLib/src/test/resources/saml.properties

```diff
@@ -0,0 +1,20 @@
+#Saml Issuer Class Name - Note that this is a compile time configuration
+org.apache.ws.security.saml.issuerClass=gov.hhs.fha.nhinc.saml.SAMLIssuerImpl
```

Comment:
Do we use this entry (issuerClass)? Verify every entires in the file to see which one we use. If we don't use, please consider to remove them